worker/uniter/uniter.go

```diff
@@ -166,7 +166,7 @@ type UniterParams struct {
 	RebootQuerier RebootQuerier
 }
 
-type NewOperationExecutorFunc func(string, operation.State, func(string) (func(), error)) (operation.Executor, error)
+type NewOperationExecutorFunc func(operation.ExecutorConfig) (operation.Executor, error)
```

Comment:
Please add comment since this is an exported type.

Reply:
done